src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,9 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        for (int i = 0; i < 3; i++) {
+            Ball ball = new Lottery().getRandomBall();
```

Comment:
you are still creating new lottery instances on each iteration. Let's create lottery instance only once before the loop. And also you can directly print the result of getRandomBall method call, so that no redundant variables are created